packages/story-editor/src/components/panels/design/layer/layer.js

```diff
@@ -350,6 +351,15 @@ function Layer({ element }) {
     })
   );
 
+  const { hasShapeMask, removeShapeMask } = useShapeMask();
+  const hasMask = hasShapeMask(element);
+  const removeMaskTitle = __('Remove Mask', 'web-stories');
+
+  let maskId = null;
+  if (hasMask) {
+    maskId = `mask-${element?.mask?.type}-${element.id}-frame`;
```

Comment:
We should move this id generation to a central location. If someone changes the id where the mask is created, this would break.

Reply:
Can add here or in a follow up PR --- where might be a good spot?

Reply:
Some utility function in e.g. `story-editor/src/utils/generateMaskId.js`?

![image](https://user-images.githubusercontent.com/21036224/173063271-51e5e9ad-ca2d-48ba-9171-f318426452a9.png)

Reply:
@merapi @barklund 

I ended up adding it to the mask package https://github.com/GoogleForCreators/web-stories-wp/pull/11618/commits/3bc21715e7b01b7491d47e1be2f665ec8e23d649 to avoid reaching out of that package.

For this PR I updated frame.js + the `shapeMaskWrapper.js` component.